packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -45,10 +44,9 @@ const BASE_CHAIN_ID = 'eip155'
 const MAX_PROVIDERS_COUNT = 100
 const TRANSACTION_CACHE_SIZE = 50
 const BLOCK_CACHE_SIZE = 50
+const V0_PROOF_TYPE = 'raw'
+const V1_PROOF_TYPE = 'f(bytes32)'
```

Comment:
Can you comment what each of these strings represent/mean?  `f(bytes32)` is pretty opaque to me (I can guess it means a function than takes 32 bytes as an input, but still not really clear what it means for that to be the "proof type" or why we chose that string)

Reply:
We should add a comment here when we've created the eip155 namespace. 

Added a follow up story here: https://linear.app/3boxlabs/issue/RES-812/add-reference-to-caip-168-namespace-registered-values